src/crimson/osd/osd.cc

```diff
@@ -854,24 +1056,95 @@ seastar::future<> OSD::consume_map(epoch_t epoch)
   });
 }
 
+
+seastar::future<Ref<PG>>
+OSD::get_pg(
+  spg_t pgid,
+  epoch_t epoch,
+  std::unique_ptr<PGCreateInfo> info) {
```

Comment:
nit, could you move the `{` down to its own line? just so my buggy builtin parser can work better. =)

Reply:
Whoops, yeah, that's the convention for .cc files.